xml2rfc/run.py

```diff
@@ -181,6 +181,8 @@ def main():
                             help='show option values and from where they come')
     help_options.add_argument('-V', '--version', action='store_true', 
                             help='display the version number and exit')
+    help_options.add_argument('--warn-bare-unicode', action='store_true', default=False,
```

Comment:
My original PR was liberating the RFC mode only.  The equivalent here would be to keep the warning in I-D mode (default=True), and disable it in RFC mode (default=False).  Of course, I'm happier with the code as it now is.

Reply:
We had a long discussion during an rpat call that convinced me that xml2rfc itself shouldn't be gating the entry of utf-8 into `<t>` for Internet-Drafts. I'm planning to give the IESG a heads up that it can happen, and we need to train authors and chairs to be aware of what the RFC Editor will ultimately allow. The RPC is building clarified language around that, and it will end up on authors.ietf.org.